sort.rb

```diff
@@ -0,0 +1,99 @@
+def insertion_sort(array)
+	n = array.size
+	i = 1
+	loop do
+		j = i
+		element_to_insert = array[i]
+		while (j > 0 && array[j-1] > element_to_insert)
+			if (element_to_insert < array[j-1])
+				array[j] = array[j-1]
+			end
+			j -= 1
+		end
+		array[j] = element_to_insert
+		i += 1
+		break if i == n
+	end
+	array
+end
+
+def bubble_sort(array)
+	n = array.size
+	i = 0
+	loop do
+		break if i == n - 1
+		j = 0
+		loop do
+			break if j == n - 1
+			if array[j] > array[j+1]
```

Comment:
To swap 2 elements, `a` and `b` in ruby - `a, b = b, a`.This will eliminate the use of `temp` varaible.